### ISSUE TITLE
Histogram on lists

### ISSUE THREAD

**Issue by dbrodie** [saulpw/visidata#712]
Right now running a histogram on lists is not very valuable at the moment is it does a histogram on the length of the list.

I would like to do a histogram based on the items in the list. For example, on this JSON data and opening (vd test.json):
[
    {
        "name": "Bob",
        "tags": ["A", "C"]
    },
    {
        "name": "Ray",
        "tags": ["A"]
    },
    {
        "name": "John",
        "tags": ["B"]
    },
    {
        "name": "Mike",
        "tags": ["C"]
    },
    {
        "name": "Jake",
        "tags": ["A", "B"]
    },
    {
        "name": "Donald",
        "tags": []
    }
]

When doing Shift-F on the tags column:
tags,count,percent,histogram
[1],3,60.00,**************************************************
[2],2,40.00,*********************************

I would like to get (pretend I drew a histogram in ascii art):
tags,count,percent,histogram
A,3,50.00,***********************************
B,2,33.33,***********
C,2,33.33,**********
<none>,1,16.66,*

So basically, we count each tag separately, and the percentage would NOT necessarily add to 100 (could be more or less). i don't see that as the problem, since the percentage would still mean what percent of items have that tag.

I would have no problem implementing it if that is something that you agree on. I could either change the current histogram code, or add a new type of histogram shortcut (since maybe someone does want the old behaviour, though it can be recreated if the behaviour changed, just add a new column that is len of the tags column).

Otherwise I could just do it as a plugin.

**Comment by saulpw** [comment]
If the lists are small, try converting the columns to string format with `~` and then getting a freq table for that.  How close does that get you to what you want?

**Comment by geekscrapy** [comment]
I would probably `unfurl-col` the tags column and then do a Freq on that column? Should give you exactly what you need.

Unfurl: #623

**Comment by saulpw** [comment]
Ah, I misread at first, and I think @geekscrapy is right.  unfurl followed by freq is what you want.

**Comment by dbrodie** [comment]
Interesting. Looks like it. It's not exactl, but I think it's close enough.
I'll close for now, thank you!

**Comment by dbrodie** [comment]
Tried it out now more fully. Works great!

An amazing way to read email (after dumping all my emails from gmail as JSON) :)
